tests/test_average.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+import pytest
+import torch
+
+from tests.helpers.testers import MetricTester, NUM_BATCHES, BATCH_SIZE
+from torchmetrics.average import AverageMeter
+
+
+def average(values, weights):
+    return np.average(values, weights=weights)
+
+
+@pytest.mark.parametrize(
+    "values, weights",
+    [
+        (torch.rand(NUM_BATCHES, BATCH_SIZE), torch.ones(NUM_BATCHES, BATCH_SIZE)),
```

Comment:
Given that you are supporting both tensors and non-tensors, better test it as well here - e.g. it should catch the `device` issue I'm pointing out when run on GPUs.